test/packetTest.js

```diff
@@ -176,7 +177,30 @@ const values = {
   tags: [{ tagName: 'hi', entries: [1, 2, 3, 4, 5] }],
   ingredient: [slotValue],
   particleData: null,
-  chunkBlockEntity: { x: 10, y: 11, z: 12, type: 25 }
+  chunkBlockEntity: { x: 10, y: 11, z: 12, type: 25 },
+  command_node: {
```

Comment:
I guess that works but it's not a very general fix

Would be much better to just propagate the recursion to the custom types